designcompose/src/main/java/com/android/designcompose/Layout.kt

```diff
@@ -325,106 +291,6 @@ internal object LayoutManager {
     }
 }
 
-private class LayoutDeltaCanvas :
```

Comment:
Pretty satisfying to see all this junk go away! Thank goodness the Compose folks added the `PlatformTextStyle` stuff :).